integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestLoadDataFrame.scala

```diff
@@ -70,7 +82,17 @@ class TestLoadDataFrame extends QueryTest with BeforeAndAfterAll {
     buildTestData
   }
 
-
+test("test the boolean data type"){
+  booldf.write
+    .format("carbondata")
+    .option("tableName", "carbon10")
+    .option("tempCSV", "true")
+    .option("compress", "true")
+    .mode(SaveMode.Overwrite)
+    .save()
+  checkAnswer(
+  sql("SELECT * FROM CARBON10"),Seq(Row("anubhav",true),Row("prince",false)))
```

Comment:
change to 
```
checkAnswer(
     sql("SELECT * FROM CARBON10"),
     Seq(Row("anubhav", true), Row("prince", false)))
```

Reply:
@jackylk i have done the changes you can review